deployments/helm-chart/Chart.yaml

```diff
@@ -1,13 +1,13 @@
 name: nginx-ingress
 version: 0.8.0
-appVersion: 1.10.0
+appVersion: 1.10.1
```

Comment:
@lucacome should the chart ver be 0.8.1 above?

Reply:
yep, nice catch! 👍